### PR TITLE
 [bc/de-de] added translation in german for bc

### PR DESCRIPTION
**Pull request by caminsha**
I resolved the issues in [this](https://github.com/adambard/learnxinyminutes-docs/pull/3633) Pull request. 

Unfortunately I could not get the source branch anymore, so I had to create a new PR. I'm sorry.

- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: de-de/bc.html.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+/* Ausserdem gibt es ++ (Inkrement) und -- (Dekrement) Operatoren */
+/* Es gibt 3 spezielle Variablen:
+scale: definiert die Anzahl Nachkommastellen
+ibase: definiert die Basis der Eingabe
+obase: definiert die Basis der Ausgabe*/
+/*Wenn-Bedingungen:*/
+hour = read() /*Eingabe einer Zahl*/
+
+if(hour < 12) { /*Operatoren sind genau wie in C*/
+    print "Guten Morgen\n" /*"print" outputs strings or variables
```
Comment (mxschumacher) [inline de-de/bc.html.markdown]: Hey @caminsha - thank you for your contribution! This line can be deleted afaict

File: de-de/bc.html.markdown [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	/*Variablen können lokal gemacht werden. Benutze das Schlüsselwort "auto" in einer Funktion.*/
+}
+
+/* Jede Variable hat als Voreinstellung den Wert 0. */
+num = blankVariable /*num wurde auf 0 gesetzt.*/
+
+/*Wie in C ist nur 0 falsch.*/
+if(!num) {print "false\n"}
+
+/*Im Gegensatz zu C hat bc die Operatoren ?: nicht. Zum Beispiel
```
Comment (mxschumacher) [inline de-de/bc.html.markdown]: ternary operator => Ternäroperator?

File: de-de/bc.html.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    print "Guten Morgen\n" /*"print" outputs strings or variables
+    print "Guten Morgen\n" /*"print" Gibt Strings oder Variablen
+	mit einem Komma separiert aus.*/
+} else if(hour == 12) {
+    print "Hallo\n"
+	/* Escape-Sequenzen starten mite einem \ in einem String.
+	Um Escape-Sequenzen klarer zu machen, ist hier eine vereinfachte
+	Liste, welche in bc funktioneren.:
+    \b: Backspace
+    \c: Wagenrücklauf (carriage return)
```
Comment (mxschumacher) [inline de-de/bc.html.markdown]: Is "Wagenrücklauf" a commonly used term? If not we could switch to 

\c: carriage return

File: de-de/bc.html.markdown (hunk 1 above)
Comment (caminsha) [reply de-de/bc.html.markdown]: Hey, Thanks. I removed that line!

File: de-de/bc.html.markdown (hunk 2 above)
Comment (caminsha) [reply de-de/bc.html.markdown]: I added the term Ternäroperator, sounds better.

File: de-de/bc.html.markdown (hunk 3 above)
Comment (caminsha) [reply de-de/bc.html.markdown]: I don't think it is very common, so I switched to `\c: carriage return`

**Comment by mxschumacher** [comment]
Thank you @caminsha

**Comment by caminsha** [comment]
> Thank you @caminsha

You're welcome! 
I'll also close PR #3633 .

**Pull request merged**